app/controllers/LegacyApiController.scala

```diff
@@ -407,11 +407,10 @@ class LegacyApiController @Inject()(annotationController: AnnotationController,
   def annotationEditV1(typ: String, id: String): Action[JsValue] = sil.SecuredAction.async(parse.json) {
     implicit request =>
       logVersioned(request)
-      val oldRequest = request.request
+      val oldRequest = request
```

Comment:
note for reviewer: the changes in this file are needed to adapt to the new silhouette api